recipes/gdal2/build.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+if [ `uname` == Darwin ]; then
+    PGFLAG=""
+    export LDFLAGS="-headerpad_max_install_names"
+#    export CC=clang
+#    export CXX=clang++
+else
+    PGFLAG="--with-pg=$PREFIX/bin/pg_config"
+fi
+
+CPPFLAGS="-I$PREFIX/include" LDFLAGS="-L$PREFIX/lib" \
+./configure --prefix=$PREFIX \
+--with-geos=$PREFIX/bin/geos-config \
```

Comment:
Is this the main difference from my version in the feedstock?  Can you add a test that ensure geos support is enable?